library/Acl/Manager.php

```diff
@@ -14,7 +14,7 @@
  *
  * Manages ACL lists in Redis Database
  */
-class RedisManager extends Adapter
+class Manager extends Adapter
```

Comment:
@rwhite27 lets extend from the phalcon incubator Redis Manger

Reply:
understood